guides/common/modules/proc_estimating-the-size-of-a-backup.adoc

```diff
@@ -18,8 +18,8 @@ A full backup requires space to store the following data:
 . Enter the `du` command to estimate the size of uncompressed directories containing {Project} database and configuration files:
 +
 ----
-# du -sh /var/lib/pgsql/data /var/lib/pulp
-100G    /var/lib/pgsql/data
+# du -sh {postgresql-data-dir} /var/lib/pulp
```

Comment:
this doesn't get replaced, you need those magic options to the codeblock :roll_eyes: